### ISSUE TITLE
Menu-searchable feature is required for keyboard users

### ISSUE THREAD

**Issue by jooyoungseo** [rstudio/rstudio#6958]
### System details

    RStudio Edition : Desktop or Server
    RStudio Version : 1.4.390
    OS Version      : any
    R Version       : any

### Steps to reproduce the problem

This is not a bug report; rather, this is a feature request.

For those who havily rely on keyboard shortcuts in their short-term memory, it would be nicer if we had a menu-searchable system just like Google docs.

Please experience with the feature implemented in G Suite:

1. Open [this example Google Doc](https://docs.google.com/document/d/1fonLshlIhMRp-ST30gmGpySiJTSC1OrZJfT6YWm3M9g/edit#heading=h.ciehmtbq98z8).
2. Press `Alt + /` (Windows); `Option + /` (Mac) to bring up search menu.
3. Type any menu name (e.g., rename; accessibility; share, etc.) in the edit box.
4. Arrow-down the suggested list and activate any of them by pressing enter key.

### Describe the problem in detail

Currently, RStudio IDE does not have menu-searchable feature that would be very helpful for keyboard users.

### Describe the behavior you expected

Google-Doc-style search menu would be ideal.

- [x] I have read the guide to submitting good bug reports at https://github.com/rstudio/rstudio/wiki/Writing-Good-Bug-Reports . 
- [x] I have installed the latest version of RStudio and confirmed that the issue still persists.
- [x] If I am reporting a RStudio crash, I have included a diagnostics report. https://support.rstudio.com/hc/en-us/articles/200321257-Running-a-Diagnostics-Report
- [x] I have done my best to include a minimal, self-contained set of instructions for consistently reproducing the issue.

**Comment by jmcphers** [comment]
Does the new Command Palette in RStudio 1.4 sufficiently address this need? You can invoke it with Ctrl + Shift + P.

**Comment by gtritchie** [comment]
Believe this is covered by the command palette feature #5168

Marking this as a duplicate. Feel free to reopen if you think this is not accurate.

**Comment by jooyoungseo** [comment]
Thanks very much @jmcphers; I did not know that! There seem some accessibility issues in the search palette. Yes, we can close this issue and re-open the accessibility-related issues separately.

**Comment by jmcphers** [comment]
Great! I'd welcome accessibility feedback on the command palette.
